ansible/slurm.yml

```diff
@@ -61,6 +61,11 @@
   tags:
     - openhpc
   tasks:
+    - include_role:
```

Comment:
Without any integration with the openhpc role, this can't actually trigger a slurm reconfigure/restart when the topology.conf file changes, is that correct? I'm kind of OK with that at least for now, but can we at least document that, given that maintenance might well move these around ...